Varianza.py

```diff
@@ -9,4 +9,64 @@
     </summary>
 """
 def menu_varianza():
-    return
\ No newline at end of file
+    print("Elige la prueba de hipotesis a utilizar:")
+    print("1.")
+    print("H0 " + Ssigma + " = " + Ssigma + "_0")
+    print("H1 " + Ssigma + " < " + Ssigma + "_0")
+    print("2.")
+    print("H0 " + Ssigma + " = " + Ssigma + "_0")
+    print("H1 " + Ssigma + " > " + Ssigma + "_0")
+    print("3.")
+    print("H0 " + Ssigma + " = " + Ssigma + "_0")
+    print("H1 " + Ssigma + " != " + Ssigma + "_0")
+    opcion = int(input("Opcion: "))
+    os.system("cls")
+    if opcion not in [1, 2, 3]:
+        print("Opcion incorrecta, vuelve a intentarlo.")
+        print("Presione cualquier tecla para continuar...")
+        os.system("pause")
+        os.system("cls")
+        menu_varianza()
+        return
+    
+    print("Prueba de hipotesis concernite a varianzas")
```

Comment:
```suggestion
    print("Prueba de hipotesis concernite a varianzas")
```
como que concernite?